tests/helpers.ts

```diff
@@ -105,4 +137,5 @@ export async function verify(fixtureName: string) {
   await install(project);
   await build(project);
   await emberTest(project);
+  await destroyProject(project);
```

Comment:
why do we need to do this?

Reply:
to cleanup temp folder

Reply:
does the OS not do that for us?

I just leave everything in /tmp 🙃

Reply:
it can grow up to few GB's (on windows), but I'm ok with removing (code part removed)

Reply:
neat -- I'm mostly just a fan of not adding code unless it solves a problem folks are having, ya know?